app/views/participant_mailer/interview_info.html.erb

```diff
@@ -0,0 +1,3 @@
+<p>
+    Hi <%= @participant.email %>, your interview is scheduled on <%= @interview.start_time %>
```

Comment:
We can use `strftime` to show time in a specific timezone as +0530

Reply:
Sir, please correct me if I'm wrong, I think strftime displays formatted date/time in stored format. After looking up some solutions, I used  .in_time_zone() to convert & display in required local format instead. 
![Screenshot from 2020-05-27 23-21-34](https://user-images.githubusercontent.com/65555640/83055461-7b622680-a071-11ea-9698-d8aeeec92683.png)

Reply:
Yes sorry.. I meant to say you can use both of them to modify the timezone and format as well.
This looks good.